src/containers/landingPage/LandingPage.jsx

```diff
@@ -1,21 +1,25 @@
 import React from 'react'
+import SearchPanel from '../../components/searchPanel/SearchPanel'
 import { TavlaLogo } from '../../assets/icons'
 import coverPhoto from '../../assets/bilde.png'
 import './styles.scss'
 
-const LandingPage = ({ addLocation }) => (
-    <div className="landing-page-wrapper">
-        <div className="title-container">
-            <img src={coverPhoto} />
-            <TavlaLogo />
-            <p>Velkommen til Entur Tavla, sanntidstavla du kan tilpasse etter dine behov.</p>
+const LandingPage = ({ addLocation }) => {
+    return (
+        <div className="landing-page-wrapper">
+            <div className="title-container">
+                <img src={coverPhoto} />
+                <TavlaLogo />
+                <p>Velkommen til Entur Tavla, sanntidstavla du kan tilpasse etter dine behov.</p>
+            </div>
+            <div className="round-shape"/>
+            <div className="content-container">
+                <SearchPanel handleCoordinatesSelected={addLocation}/>
+                <p>For å kunne opprette en tavle,  trenger vi å vite hvor du er.</p>
```

Comment:
Dobbelt mellomrom etter komma: `tavle,  trenger`.

Litt rar formulering også: "For å kunne opprette en tavle,  trenger vi å vite hvor du er." Dersom man skriv inn koordinatar sjølv, eller brukar søket, så treng me vel ikkje å vite kor du er?

Reply:
Tja, det er vel mest at vi må vite hvor du skal bruke den? Type enten adresse eller sted

Reply:
Ja, då ville eg nok omformulert litt. F.eks. "For å kunne opprette en tavle, trenger vi en posisjon å hente stoppesteder rundt." eller  "For å kunne opprette en tavle, trenger vi å vite hvilket område du er interessert i."

Reply:
Ja Cecilie var litt i tankeboblen på den der.. Vi ble enige om at det siste forslaget ditt klang bra!